docs/mixin/alerts/alerts.yaml

```diff
@@ -342,3 +342,18 @@ groups:
       summary: High uncompressed data.
       description: "High uncompressed data in Promscale, on average, {{ $value }} uncompressed chunks per metric."
       runbook_url: https://github.com/timescale/promscale/blob/master/docs/runbooks/PromscaleCompressionLow.md
+  - alert: PromscalePostgreSQLSharedBuffersLow
+    expr: |
+      (
+        ((promscale_sql_database_open_chunks_total_table_size + promscale_sql_database_open_chunks_total_index_size) 
+        / 
+        promscale_sql_database_shared_buffers_size)
+        > 1 )
+    for: 1m
```

Comment:
How do we ensure this expression is not going to be flaky due to frequent changes in this ratio?

Reply:
Is it possible to have ratio fluctuating near 1 (few minutes above 1, few minutes below)? Because if it is, the alert is flaky.

Reply:
As it is now it needs to be above 1 for 10 minutes. To me that doesn't seem flaky. What are you suggesting?